src/DataCollection/parse_csv.py

```diff
@@ -0,0 +1,57 @@
+"""
+run this to update the airports_clean data to add to airports.ts
+
+airport data is from https://gist.github.com/tdreyno/4278655 and the American Airlines destinations
+are filtered according to https://en.wikipedia.org/wiki/List_of_American_Airlines_destinations
+"""
+import json
+import wikipedia
```

Comment:
Where does this dependency come from? Can you add some notes about how to install it?

Reply:
Added information in the readme and some notes on possible errors in the script's header!